packages/voorbeeld-design-tokens/documentation/theme-builder/theme-builder-step1.stories.tsx

```diff
@@ -0,0 +1,127 @@
+// import { cssVariable } from '@nl-design-system-unstable/theme-toolkit/src/util';
+import { fontFamilystep as voorbeeldfontfamilystep } from './voorbeeld-theme';
+import type { Meta, StoryObj } from '@storybook/react';
+import {
+  Button,
+  ButtonGroup,
+  Heading1,
+  Heading2,
+  Heading3,
+  Heading4,
+  Heading5,
+  Heading6,
+  Link,
+  Paragraph,
+  NavBar,
+} from '@utrecht/component-library-react/dist/css-module';
+import { PropsWithChildren } from 'react';
+
+interface WrapperProps {
+  cssVariables: { [index: string]: any };
+}
+const Wrapper = ({ children, cssVariables }: PropsWithChildren<WrapperProps>) => (
+  <div className="voorbeeld-theme">
+    <div style={cssVariables}>{children}</div>
+  </div>
+);
+
+interface BorderRadiusStoryProps {
+  cssVariables: { [index: string]: any };
+}
+const BorderRadiusStory = ({ ...restProps }: PropsWithChildren<BorderRadiusStoryProps>) => (
+  <Wrapper {...restProps}>
+    <Button appearance="primary-action">Default</Button>
+  </Wrapper>
+);
+
+interface FontFamilyStoryProps {
+  cssVariables: { [index: string]: any };
+}
+const FontFamilyStory = ({ ...restProps }: PropsWithChildren<FontFamilyStoryProps>) => (
+  <Wrapper {...restProps}>
+    <Heading1>The quick brown fox jumps over the lazy fox</Heading1>
+    <Heading2>The quick brown fox jumps over the lazy fox</Heading2>
+    <Heading3>The quick brown fox jumps over the lazy fox</Heading3>
+    <Heading4>The quick brown fox jumps over the lazy fox</Heading4>
+    <Heading5>The quick brown fox jumps over the lazy fox</Heading5>
+    <Heading6>The quick brown fox jumps over the lazy fox</Heading6>
+    <Paragraph>The quick brown fox jumps over the lazy fox</Paragraph>
+    <ButtonGroup>
+      <Button appearance="primary-action">Default</Button>
+      <Button appearance="primary-action">Default</Button>
+    </ButtonGroup>
+    <Link href="https://www.example.com">Read more</Link>
+    <NavBar>Default</NavBar>
+  </Wrapper>
+);
+
+const meta = {
+  id: 'theme-builder-step1',
+  title: 'Theme builder/Step #1 Font Familiy',
+  component: BorderRadiusStory,
+  argTypes: {},
+  args: {},
+} satisfies Meta<typeof BorderRadiusStory>;
+
+type Story = StoryObj<typeof meta>;
+
+export default meta;
+
+export const Step1: Story = {
```

Comment:
Misschien een iets duidelijkere naam?

`export const Step1RijksHuisstijl`?

Reply:
Bij de Steps staan indd geen duidelijke beschrijvingen maar bij de `name` wel en daarna nog een description. Is dat niet genoeg of toch ook bij de stappen een title toevoegen?


```
export const Step1: Story = {
  name: 'Thema 1: Rijkshuisstijl',
  args: {
    cssVariables: {
      ...rijkshuisstijlborderradiusstep,
    },
  },
  parameters: {
    docs: {
      description: `Pas de border radius aan volgens de Rijkshuisstijl thema`,
    },
  },
};
```